gen/field.go

```diff
@@ -28,6 +28,25 @@ type fieldGroupGenerator struct {
 	Fields compile.FieldGroup
 }
 
+func (f fieldGroupGenerator) Generate(g Generator) error {
+	if err := f.DefineStruct(g); err != nil {
+		return err
+	}
+
+	if err := f.ToWire(g); err != nil {
+		return err
+	}
+
+	if err := f.FromWire(g); err != nil {
+		return err
+	}
+
+	if err := f.String(g); err != nil {
+		return err
+	}
+	return nil
```

Comment:
nit: blank before this (to match the blank between the other groups in this method)